README.md

```diff
@@ -40,15 +40,21 @@ extra seed data required.**
 
 ---
 
+
+### Set up database
+
 Setup the database connection:
 
 ```sh
 $ cp config/example.database.yml config/database.yml
 ```
-
 Be sure to remove or modify the `username` attribute.
 
-Make sure Elasticsearch is running.
+Download a backup of the Production DB and load it into your local DB. Follow the instructions from step 4 in this wiki: https://moneyadviceserviceuk.atlassian.net/wiki/display/RRAD/FCA+Import+-+Technical+Help
```

Comment:
I recommend we refer people to [Load into your local development database
](https://moneyadviceserviceuk.atlassian.net/wiki/pages/viewpage.action?pageId=63635527#DatabaseTasks(PostgreSQL)-Loadintoyourlocaldevelopmentdatabase) as it contains only the steps required to achieve the goal whereas step 4 from the import process requires the reader to have enough background knowledge to filter out the actions that are not required.